From: Anthony Fok <foka@debian.org>
Date: Sun, 22 Sep 2013 17:26:01 +0800
Subject: Clean leftover __pycache__ directories

Makefile.am: Delete src/__pycache__/ and tests/__pycache__/ during clean
---
 Makefile.am | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Makefile.am b/Makefile.am
index 96964b7..d1b5e57 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -120,6 +120,10 @@ CLEANFILES = \
 	tests/run_tests \
 	$(NULL)
 
+clean-local:
+	-rm -rf src/__pycache__
+	-rm -rf tests/__pycache__
+
 EXTRA_DIST = \
 	autogen.sh \
 	data/setup.ui \
